packages/ag-grid-theme/stories/ag-grid-theme.stories.tsx

```diff
@@ -1,8 +1,7 @@
 import { AgGridReact } from "ag-grid-react";
-import "ag-grid-community/dist/styles/ag-grid.css";
-import "ag-grid-community/dist/styles/ag-theme-material.css";
-import "../uitk-ag-theme.css";
-import "../salt-ag-theme.css";
+import "ag-grid-community/styles/ag-grid.css";
```

Comment:
Is this needed? if so, usage doc needs update

Reply:
Yes. We apply our styles on top of the ag grid styles. Will update docs in the next PR, there will be a lot of changes there